aspose/pom.xml

```diff
@@ -25,13 +25,6 @@
 			<artifactId>lombok</artifactId>
 		</dependency>
 
-		<!-- Explicit dependency to mitigate CVE-2022-3171 -->
-		<dependency>
-			<groupId>com.google.protobuf</groupId>
-			<artifactId>protobuf-java</artifactId>
-			<version>3.24.3</version>
-		</dependency>
```

Comment:
Should be moved to parent pom, since somewhere else there is still a transitive dep found, using the older version...